Tasks/DotNetCoreCLIV2/dotnetcore.ts

```diff
@@ -2,6 +2,7 @@ import tl = require("azure-pipelines-task-lib/task");
 import tr = require("azure-pipelines-task-lib/toolrunner");
 import path = require("path");
 import fs = require("fs");
+import ltx = require("ltx");
```

Comment:
dont you need to declare this in package.json

Reply:
ltx is part of packaging-common which is already there in package.json